parts/linux/cloud-init/artifacts/update_certs.sh

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/env bash
 set -uo pipefail
-set -x
```

Comment:
so _no_ logging? I might slap some echoes in the iterations where we actually do work?